Shared/sdk/net/rpc_enums.h

```diff
@@ -271,6 +271,7 @@ enum eElementRPCFunctions
   
     SET_DISCORD_JOIN_PARAMETERS,
     SET_TRANSFER_BOX_VISIBILITY,
+    COLSHAPE_POLYGON_SET_HEIGHT,
```

Comment:
Perhaps rename to `SET_COLPOLYGON_HEIGHT` for consistency.
```suggestion
    SET_COLPOLYGON_HEIGHT,
```